build.gradle

```diff
@@ -2803,6 +2793,7 @@ project(':streams:streams-scala') {
     testImplementation project(':server-common').sourceSets.test.output
     testImplementation project(':streams').sourceSets.test.output
     testImplementation project(':clients').sourceSets.test.output
+    testImplementation project(':streams:integration-tests').sourceSets.test.output
```

Comment:
`streams` module can't depend on `integration-tests` as all integration tests should be moved from `stream` to `integration-tests` already.

Reply:
Updated it. The failed case will be handled in https://github.com/apache/kafka/pull/17645.